src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -120,14 +126,19 @@ public virtual void WriteMessage(LogLevel logLevel, string logName, int eventId,
                 logLevelColors = GetLogLevelConsoleColors(logLevel);
                 logLevelString = GetLogLevelString(logLevel);
                 // category and event id
-                logIdentifier = _loglevelPadding + logName + "[" + eventId + "]";
+                _sb.Append(_loglevelPadding);
+                _sb.Append(logName);
+                _sb.Append("[");
+                _sb.Append(eventId);
+                _sb.AppendLine("]");
                 // scope information
                 if (IncludeScopes)
                 {
-                    scopeInformation = GetScopeInformation();
+                    _sb.AppendLine(GetScopeInformation());
```

Comment:
`GetScopeInformation` should be passed the current string builder rather creating a new one